README.md

```diff
@@ -475,6 +476,19 @@ These behave exactly like VueJs's transition directives, except they have differ
 
 ---
 
+### `x-ignore`
+**Example:**
+```html
+<div x-ignore="true">
+    <div x-data="{foo:'bar'}">
+        <span x-text="foo">"foo" never gets bind by alpine</span>
```

Comment:
`bind` should probably be `bound`

Also `alpine` -> `Alpine` (this one is a total nitpick, feel free to ignore)

Reply:
- Done